### PR TITLE
apiextensions-apiserver: fix empty APIResource group+version handling in wait logic

### PR DESCRIPTION
**Pull request by sttts**
Also get rid of wrong upstream carry.

Compare https://github.com/kubernetes/kubernetes/pull/74587#issuecomment-468183531.

### CODE REVIEW

File: vendor/k8s.io/kubernetes/staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go [hunk 1]
```diff
@@ -276,6 +278,16 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
...
 }
 
+func splitGroupVersion(gv string) (group string, version string) {
+	ss := strings.SplitN(gv, "/", 2)
+	if len(ss) == 1 {
+		version = ss[0]
+	} else {
+		group, version = ss[0], ss[1]
+	}
+	return
```
Comment (soltysh) [inline vendor/k8s.io/kubernetes/staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go]: Not needed.

**Comment by soltysh** [comment]
/lgtm cancel
the 2nd commits is not correct, still

**Comment by sttts** [comment]
@deads2k please update your upstream PR.

**Pull request merged**